### PR TITLE
feat(VSelect): replicate html select hotkeys

### LINKED ISSUE #4853: [Feature Request] Replicate HTML Select hotkeys in Vuetify <v-select>
### Problem to solve
In a vanilla html select input, once the input is focused, the different options can be quick-selected by typing the first letter of an option. This not only highlights the value, but also  **selects** that value.

This may seem trivial, but it is a standardized feature in select inputs, and is an important feature for semi-advanced users who are accustomed to faster data input methods.

This pen shows the issue:
https://codepen.io/smcolburn/pen/zLbXOy?&editors=101

To reproduce:
1. Click on the vuetify `<v-select>` input.
2. Press `F`. Nothing happens.
3. Click on the HTML `<select>` input.
3. Press `D`. `Dog` is automatically highlighted. If the user tabs away, `Dog` is saved as the input value.

### Proposed solution
`<v-select>` should emulate standard HTML inputs in this regard. If a user begins to type with the input focused, it should automatically select the closest matched value to their input based on the `itemValue` prop, string match, or possibly a filter function. 

**THIS IS NOT THE SAME AS V-AUTOCOMPLETE** 
Note that is not the same as `v-autocomplete` or the basic filtering functionality therein. The autocomplete suggests options based on the users input, and filters possible values based on that input.

This is recreating the standard HTML select functionality. No options should be filtered out, and the closest matching option should be automatically selected from the list of options.

### PR DESCRIPTION
**Pull request by dima74**
## Description

Possible implementation for quick select item in `v-select` by typing first few letters of item

## Motivation and Context

Fixes #4853

## How Has This Been Tested?

visually & unit

## Markup:

<details>

```vue
<template>
  <v-app>
    <v-layout align-center justify-center>
      <v-flex xs4 text-xs-center>

        <v-text-field solo />
        <v-select
          solo
          :items="items"
          label="Standard"
        />
        <a href="https://codepen.io/dima74/pen/yGXabW" target="_blank">codepen with native select</a>

      </v-flex>
    </v-layout>
  </v-app>
</template>

<script>
  export default {
    data: () => ({
      items: ['Aaa', 'Ooo', 'Faa', 'Foo'],
    }),
  };
</script>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VSelect/VSelect.js [hunk 1]
```diff
@@ -387,6 +389,7 @@ export default VTextField.extend({
       input.data.domProps.value = null
       input.data.attrs.readonly = true
       input.data.attrs['aria-readonly'] = String(this.readonly)
+      input.data.on.keypress = this.onKeyPress
```
Comment (KaelWD) [inline packages/vuetify/src/components/VSelect/VSelect.js]: keypress is deprecated, move to the keydown handler

File: packages/vuetify/src/components/VSelect/VSelect.js [hunk 2]
```diff
@@ -564,6 +567,22 @@ export default VTextField.extend({
         this.isMenuActive = false
       }
     },
+    onKeyPress (e) {
+      if (this.multiple) return
+
+      const KEYBOARD_LOOKUP_THRESHOLD = 1000 // milliseconds
+      const now = Date.now()
```
[the remark this answers is not in the document]
Comment (dima74) [reply packages/vuetify/src/components/VSelect/VSelect.js]: Yes, thank you

File: packages/vuetify/src/components/VSelect/VSelect.js (hunk 1 above)
Comment (dima74) [reply packages/vuetify/src/components/VSelect/VSelect.js]: ok, i will change it soon

Comment (dima74) [reply packages/vuetify/src/components/VSelect/VSelect.js]: @KaelWD so I tried to use keydown handler, but I couldn't find good way to detect if pressed key is printable (https://stackoverflow.com/q/43877434/). Looks like proper solution would be to use [`beforeinput`](https://developer.mozilla.org/en-US/docs/Web/Events/beforeinput) event, but it has poor browser support ([not implemented in Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=970802)). Maybe it is good idea to keep `keypress`...

Comment (KaelWD) [reply packages/vuetify/src/components/VSelect/VSelect.js]: That's a bit silly, guess it'll have to do then.

**Pull request merged**